pywhy_graphs/export/tetrad.py

```diff
@@ -0,0 +1,157 @@
+from typing import Dict
+
+import pywhy_graphs
+from pywhy_graphs.config import TetradEndpoint
+
+
+def tetrad_to_graph(filename: str, graph_type):
+    """Convert a tetrad stored graph from a text file to causal graph in pywhy.
+
+    Parameters
+    ----------
+    filename : str
+        The file at which the tetrad file is stored.
+    graph_type : str
+        Type of pywhy causal graph to construct.
+        One of ('pag', 'cpdag', 'admg', 'dag').
+
+    Returns
+    -------
+    G : instance of causal graph
+        The causal graph.
+
+    Notes
+    -----
+    See tetrad documentation: https://cmu-phil.github.io/tetrad/manual/.
+
+    Warning: There is no guarantee or check that your file adheres to the
+    tetrad format.
+    """
+    # instantiate the type of causal graph
+    if graph_type == "dag":
```

Comment:
I think I have seen different ways of dealing with DAGs - should we use nx.DiGraph or send it to pywhy_graphs? ADMG here seems like a hack. Maybe an in-house pywhy_graphs DAG class is needed?

Reply:
True. Perhaps `nx.DiGraph` with some added semantics to ensure construction is acyclic... But it'd be too expensive to check acyclicity every time someone adds a node. 

On one hand it seems overkill to recreate a DAG class when nx.DiGraph works. On the other hand, the mental switching is a bit weird. 

Tbh not sure exactly which way to go.